core/src/main/java/com/spotify/metrics/jvm/FileDescriptorGaugeSet.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2017 Spotify AB.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.metrics.jvm;
+
+import com.codahale.metrics.Gauge;
+import com.codahale.metrics.Metric;
+import com.codahale.metrics.jvm.FileDescriptorRatioGauge;
+import com.spotify.metrics.core.MetricId;
+import com.spotify.metrics.core.SemanticMetricSet;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * A set of gauges for the ratio of allocated file descriptors.
+ */
+public class FileDescriptorGaugeSet implements SemanticMetricSet {
+
+    private FileDescriptorRatioGauge fileDescriptorRatioGauge;
+
+    public FileDescriptorGaugeSet() {
+        fileDescriptorRatioGauge = new FileDescriptorRatioGauge();
+    }
+
+    @Override
+    public Map<MetricId, Metric> getMetrics() {
+        fileDescriptorRatioGauge.getValue();
+        final Map<MetricId, Metric> gauges = new HashMap<MetricId, Metric>();
+        final MetricId metricId =
+                MetricId.build().tagged("what", "file-descriptor-ratio", "unit", "ratio");
```

Comment:
As to your question about the unit: I believe that "%" would be reasonable. It's used in other places and fits well with a value in the range of [0, 1].